EZSwiftExtensionsTests/StringTests.swift

```diff
@@ -146,8 +146,8 @@ class StringTests: XCTestCase {
 
     func testContains() {
         XCTAssertTrue(string.contains("01"))
-        XCTAssertTrue(string.contains("01", compareOption: NSStringCompareOptions.AnchoredSearch))
-        XCTAssertFalse(string.contains("12", compareOption: NSStringCompareOptions.AnchoredSearch))
+        XCTAssertTrue(string.contains("01", compareOption: NSString.CompareOptions.anchored))
```

Comment:
Doesn't really matter in this case, sometimes it might be useful to see the full thing too.